pkg/monitoring/rules/alerts/operator_alerts.go

```diff
@@ -0,0 +1,59 @@
+package alerts
+
+import (
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+)
+
+const (
+	outOfBandUpdateAlert          = "KubeVirtCRModified"
+	unsafeModificationAlert       = "UnsupportedHCOModification"
+	installationNotCompletedAlert = "HCOInstallationIncomplete"
+	singleStackIPv6Alert          = "SingleStackIPv6Unsupported"
+	severityAlertLabelKey         = "severity"
+	healthImpactAlertLabelKey     = "operator_health_impact"
+)
+
+func createRule(alert, expr, description, summary, severity, healthImpact string, duration *promv1.Duration) promv1.Rule {
+	return promv1.Rule{
+		Alert:       alert,
+		Expr:        intstr.FromString(expr),
+		For:         duration,
+		Annotations: map[string]string{"description": description, "summary": summary},
+		Labels:      map[string]string{severityAlertLabelKey: severity, healthImpactAlertLabelKey: healthImpact},
+	}
+}
```

Comment:
I think using a function here, although saving a few characters make code less readable

Reply:
sure? that way you can also edit the label for all and see all available fields

Reply:
and also make sure if someone adds a new alert he will include all fields

Reply:
doesn't prevent an empty value, for example, that's why we have the linters
and you can't really edit for all, since it's all arguments

I prefer to have the promv1.Rule inline